include/itkFFTWCommonExtended.h

```diff
@@ -45,8 +45,8 @@ class ComplexToComplexProxy
 {
   // empty -- only double and float specializations work
 protected:
-  ComplexToComplexProxy(){};
-  ~ComplexToComplexProxy(){};
+  ComplexToComplexProxy() = default;
+  ~ComplexToComplexProxy() override = default;
```

Comment:
[CDash result](include/itkFFTWCommonExtended.h:49:28: error: only virtual member functions can be marked 'override')

Looks like `ComplexToComplexProxy` does not inherit from a superclass and should not use `override`